docs/docs/cli/generated/tracetest_test_list.md

```diff
@@ -0,0 +1,33 @@
+# CLI Referece
+## tracetest test list
+
+list all test
+
+### Synopsis
+
+list all test
+
+```
+tracetest test list [flags]
+```
+
+### Options
```

Comment:
In the future: should we add a search/filter option here?

Reply:
yes, probably we should allow it